src/client/idv.client.js

```diff
@@ -93,6 +93,28 @@ class IDVClient {
     return this.idvService.getSupportedDocuments();
   }
 
+  /**
+   * @param {string} sessionId
+   * @param {CreateFaceCaptureResourcePayload} createFaceCaptureResourcePayload
+   *
+   * @returns {Promise<CreateFaceCaptureResourceResponse>}
+   */
+  createFaceCaptureResource(sessionId, createFaceCaptureResourcePayload) {
+    return this.idvService.createFaceCaptureResource(sessionId, createFaceCaptureResourcePayload);
+  }
+
+  /**
+   * @param {string} sessionId
+   * @param {string} resourceId
+   * @param {UploadFaceCaptureImagePayload} uploadFaceCaptureImagePayload
+   *
+   * @returns {Promise<CreateFaceCaptureResourceResponse>}
+   */
+  uploadFaceCaptureImage(sessionId, resourceId, uploadFaceCaptureImagePayload) {
+    // eslint-disable-next-line max-len
```

Comment:
minor:

Just wrap the arguments onto separate lines to avoid the `eslint-disable`